docs/content/manual/pre-release/resiliency/pvc_provisioning_with_insufficient_storage.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "PVC provisioning with insufficient storage"
+---
+
+##### Related Issue:
+- https://github.com/longhorn/longhorn/issues/4654
```

Comment:
Suggest adding https://github.com/longhorn/longhorn/issues/4654#issuecomment-1264870672 here to make a RCA (root cause analysis) shortcut.

Reply:
Hi @innobead , I added RCA for review, thank you.